packages/pie-toolbox/src/code/editable-html/editor.jsx

```diff
@@ -605,6 +605,11 @@ export class Editor extends React.Component {
     // Check if relatedTarget is a done button
     const isRawDoneButton = relatedTarget?.closest('button[class*="RawDoneButton"]');
```

Comment:
I don't think this will work in a deployed environment. Because the classes are built there and they have a different name. Please handle it similarly to how i did it here:
https://github.com/pie-framework/pie-lib/blob/9d0d0ee7667bebeebb4344c3a768dc1033411b87/packages/pie-toolbox/src/code/editable-html/editor.jsx#L603

Reply:
@lakatosandrei thanks for your guidance on the class names issue. I've updated the code accordingly. Can you check if everything looks good now?

Reply:
This looks good in my opinion. But @andreeapescar suggested a cleaner approach.
Using classes and querying the elements that way. But this would require adding an additional class to the elements you want to target.